### PR TITLE
Avalanche Support for Human Protocol

### PR DESCRIPTION
**Pull request by hrik2001**
## Description

PR that aims to add support of Avalanche mainnet and testnet, as discussed in #137 

## Summary of changes

Added network details to `escrow-dashboard` and `CONTRACT_LIST.md`

### CODE REVIEW

**Comment by posix4e** [comment]
Where is fortune running?

**Comment by hrik2001** [comment]
@posix4e I had deployed the fortune app locally, that ran on
- Fujinet (Avalanche testnet)
- Avalanche C-Chain

[Testnet Video Demo](https://www.loom.com/share/f32cc0e4ca8748ccbb654b81920591a8)
[Mainnet Video Demo](https://www.loom.com/share/50883365395c43bc9cdf7c83c1394a24)

**Comment by foufrix** [comment]
@posix4e everything is on this message on this avalanche issue : https://github.com/humanprotocol/human-protocol/issues/137#issuecomment-1372085275

**Review by foufrix — APPROVED** [review]

**Review by foufrix — CHANGES_REQUESTED** [review]
The previous change that you published on the previous repo @hrik2001 (the config for deployment here : https://github.com/humanprotocol/hmt-escrow/pull/430/files#diff-5f1179b38d94a75785290064a194e2fd668a28ccbb30b8784d1cc34bafeb35f6) need to be added to `hardhat.config.ts` file in `core` https://github.com/humanprotocol/human-protocol/blob/main/packages/core/hardhat.config.ts

Apart from that looks good to me

**Comment by portuu3** [comment]
@hrik2001 please verify the contracts in both testnet and mainnet scanners

File: CONTRACTS_LIST.md [hunk 1]
```diff
@@ -8,4 +8,6 @@
 | Rinkeby | Testnet | Deprecated |0x4dCf5ac4509888714dd43A5cCc46d7ab389D9c23 |  | 0x925B24444511c86F4d4E63141D8Be0A025E2dca4 | | |
 | Binance Smart Chain  | Active | Mainnet | 0x0d501B743F22b641B8C8dfe00F1AAb881D57DDC7 |  | 0xc88bC422cAAb2ac8812de03176402dbcA09533f4 | | |
 | Binance Smart Chain | Active | Testnet | 0xd3a31d57fdd790725d0f6b78095f62e8cd4ab317 |  | 0xaae6a2646c1f88763e62e0cd08ad050ea66ac46f | | |
-| Moonbeam | Mainnet | Active | 0x3b25BC1dC591D24d60560d0135D6750A561D4764 |  | 0x98108c28B7767a52BE38B4860832dd4e11A7ecad | | |
\ No newline at end of file
+| Moonbeam | Mainnet | Active | 0x3b25BC1dC591D24d60560d0135D6750A561D4764 |  | 0x98108c28B7767a52BE38B4860832dd4e11A7ecad | | |
+| Fuji C-Chain | Testnet | Active | 0x9406d5c635AD22b0d76c75E52De57A2177919ca3 |  | 0xfb4469201951C3B9a7F1996c477cb7BDBEcE0A88 | | |
+| Moonbeam | Mainnet | Active | 0x12365293cb6477d4fc2686e46BB97E3Fb64f1550 |  | 0x9767a578ba7a5FA1563c8229943cB01cd8446BB4 | | |
```
Comment (portuu3) [inline CONTRACTS_LIST.md:13]: I think should be Avalanche instead of Moonbeam here

**Review by portuu3 — CHANGES_REQUESTED** [review]

**Comment by portuu3** [comment]
Fortune works for both testnet and mainnet. Waiting for changes requested

**Comment by hrik2001** [comment]
Hi, thanks @foufrix @portuu3, had to run some errands hence couldn't do the changes. Changes will be done in an hour , thanks a lot :)

**Comment by portuu3** [comment]
@hrik2001 contracts still not verified

**Review by foufrix — DISMISSED** [review]

**Comment by portuu3** [comment]
@foufrix I cannot be approved and merged yet, if we merge now we will break the dashboard since the subgraphs are not done.
@hrik2001 please can you implement subgraphs too?

**Comment by hrik2001** [comment]
@portuu3 sure, so essentially I have to add configs for avalanche at `human-protocol/packages/sdk/typescript/subgraph/config/` right?

Also you had said
> contracts still not verified

I am not sure what you mean by this, how should I go about that?

**Comment by portuu3** [comment]
> Also you had said
> 
> > contracts still not verified
> 
> I am not sure what you mean by this, how should I go about that?

If you click in the contract tab of the scanner you will have an option to verify it. Just click on it and go trough the process, it will ask you to upload the source code of the contracts you used.
image

**Comment by portuu3** [comment]
> @portuu3 sure, so essentially I have to add configs for avalanche at `human-protocol/packages/sdk/typescript/subgraph/config/` right?

The section `Add a new network` in `human-protocol/packages/sdk/typescript/subgraph/README.md` should help you

**Comment by hrik2001** [comment]
@portuu3 just wanted to confirm, if I host the subgraph, there would be my username in the subgraph link and not human protocol, you guys are okay with that right?

**Comment by portuu3** [comment]
> @portuu3 just wanted to confirm, if I host the subgraph, there would be my username in the subgraph link and not human protocol, you guys are okay with that right?

@hrik2001 just do the changes in the repo and I can do the step for creating the subgraph

**Comment by hrik2001** [comment]
@portuu3 feel free to review

**Review by portuu3 — APPROVED** [review]

**Pull request merged**